specs/agents/tracing-instrumentation-aws-lambda.md

```diff
@@ -132,6 +132,66 @@ In version 2.0, the `${event.requestContext.routeKey}` can have the format `GET
 
 If `use_path_as_transaction_name` is applicable and set to `true`, use `${event.requestContext.http.method} ${event.requestContext.http.path}` as the transaction name.
 
+### Elastic Load Balancer (ELB)
+
+Elastic Load Balancer (ELB) can be attached directly to lambda, without the use
+of API Gateway. In this case the `event` object will be structured differently.
```

Comment:
Not for this PR, but we could also consider differentiating [the relatively new Lambda Function URLs](https://aws.amazon.com/blogs/aws/announcing-aws-lambda-function-urls-built-in-https-endpoints-for-single-function-microservices/) ([docs](https://docs.aws.amazon.com/lambda/latest/dg/lambda-urls.html), [payload examples](https://docs.aws.amazon.com/lambda/latest/dg/urls-invocation.html#urls-payloads)) from API Gateway-triggered Lambda invocations at some point.

Reply:
Eh, I might just make that part of this spec change. Might as well as long as I'm here.

Reply:
Actually, on inspecting the [documentation you linked](https://docs.aws.amazon.com/lambda/latest/dg/urls-invocation.html#urls-payloads) it appears to follow the APIv2 schema exactly, so I think we can safely treat it as the same trigger type.

If we care about differentiating the trigger, we could detect the difference by searching the url for `.lambda-url.` and fill `faas.trigger.type` accordingly:

```
https://<url-id>.lambda-url.<region>.on.aws
```

But I honestly think `http` fits lambda urls well enough.

Reply:
Definitely `faas.trigger.type = "http"` works.
Possibly we'd want a different value for `context.cloud.origin.service.name = 'api gateway'`, say `'lambda url'`.

Reply:
Done.